content/syllabuses/k8s.md

```diff
@@ -33,4 +33,11 @@ title: Kubernetes (A.K.A K8s)
 
 - {{< contentlink path="k8s/kubernetes/deploy-app-with-manifest-file/install-postgresql" >}}
 - {{< contentlink path="k8s/kubernetes/deploy-app-with-manifest-file/deploy-frontend" >}}
-- {{< contentlink path="k8s/kubernetes/deploy-app-with-manifest-file/deploy-backend" >}}
\ No newline at end of file
+- {{< contentlink path="k8s/kubernetes/deploy-app-with-manifest-file/deploy-backend" >}}
+
+- {{< contentlink path="k8s/helm-deployment" >}}
```

Comment:
This topic is empty - should there be something in there? Or should it be excluded?

Reply:
It is supposed to just be a introductory text, which was missing. We might add more information there in the future.